core/frontend/src/main/scala/io/udash/properties/Properties.scala

```diff
@@ -2,6 +2,7 @@ package io.udash.properties
 
 trait Properties {
   implicit val allowDomElementTpe: ImmutableValue[org.scalajs.dom.Element] = null
+  implicit val allowJavaDateTpe: ImmutableValue[java.util.Date] = null
```

Comment:
Yes, but we assume it is. :D 

I want to use `ju.Date` in datepicker API and we need this assumption to create `Property[ju.Date]`. Mutable API of `ju.Date` is deprecated, so it is almost immutable. ;)